CHANGELOG.md

```diff
@@ -1,5 +1,49 @@
 # Changelog for Elixir v1.9
 
+## Releases
+
+The main feature in Elixir v1.9 is the addition of releases. A release is a self-contained directory with your application code, all of its dependencies, plus the whole Erlang VM and runtime. Once a release is assembled, it can be packaged and deployed to a target, as long as the target runs on the same operating system (OS) distribution and version as the machine running the `mix release` command.
+
+You can start a new project and assemble a release for it in 3 easy steps:
+
+    $ mix new my_app
+    $ cd my_app
+    $ MIX_ENV=prod mix release
+
+A release will be assembled in `_build/prod/rel/my_app`. Inside the release, there will be a `bin/my_app` file which is the front-end to your system. It supports multiple commands, such as:
+
+  * `bin/my_app start`, `bin/my_app start_iex`, `bin/my_app restart`, and `bin/my_app stop` - for general management of the release
+
+  * `bin/my_app rpc COMMAND` and `bin/my_app remote` - for running commands on the running system or to connect to the running system
+
+  * `bin/my_app eval COMMAND` - to start a fresh system that runs a single command and then shuts down
+
+  * `bin/my_app daemon` and `bin/my_app daemon_iex` - to start the system as a daemon on Unix-like systems
+
+  * `bin/my_app install` - to install the system as a service on Windows machines
+
+Releases also provide built-in hooks for configuring almost every need of the production system:
+
+  * `config/config.exs` (and `config/prod.exs`) - provides build-time application configuration. Those are executed when the release is assembled
+
+  * `config/releases.exs` - provides runtime application configuration. It is executed every time the release boots and is further extensible via config providers
+
+  * `rel/vm.args.eex` - a template file that is copied into every release and provides static configuration of the Erlang Virtual Machine and other runtime flags
+
+  * `rel/env.sh.eex` and `rel/env.bat.eex` - a template file that is copied into every release and is executed on every command to set up environment variables, including VM specific env vars, and the general environment
+
+We have written extensive documentation on releases, so we recommend checking it out for more information.
+
+## Configuration overhaul
+
+A new `Config` module has been added to Elixir. The previous configuration API, `Mix.Config`, was part of the Mix build tool. But since releases provide runtime configuration and Mix is not included in releases, we ported the Config API to Elixir. In other words, `use Mix.Config` has been soft-deprecated in favor of `import Config`.
```

Comment:
```suggestion
A new `Config` module has been added to Elixir. The previous configuration API, `Mix.Config`, was part of the Mix build tool. But since releases provide runtime configuration and Mix is not included in releases, we ported the `Mix.Config` API to Elixir. In other words, `use Mix.Config` has been soft-deprecated in favor of `import Config`.
```